akka-http-caching/src/main/mima-filters/10.1.7.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# Scala 2.13.0-M5
+ProblemFilters.exclude[MissingTypesProblem]("akka.http.caching.scaladsl.CachingSettings$")
```

Comment:
What happened here?

Reply:
I think that settings class doesn't seem to follow the usual pattern anyway. Shouldn't it derive from the public `akka.http.scaladsl.settings.SettingsCompanion`? Not really related to this one here. So, let's create a ticket for that and fix that afterwards.